src/library/scala/math/Equiv.scala

```diff
@@ -35,9 +35,11 @@ trait Equiv[T] extends Any with Serializable {
   def equiv(x: T, y: T): Boolean
 }
 
+@deprecated("This only existed for the implicit universalEquiv. Use the explicit Equiv.universal instead.", "2.13.0")
```

Comment:
As Miles says, given the self-type, this isn't useable anywhere outside of `Equiv`, so let's avoid the spam.